inc/class-sitemaps-categories.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * Class Core_Sitemaps_Categories.
+ * Builds the sitemap pages for Categories.
+ */
+class Core_Sitemaps_Categories extends Core_Sitemaps_Provider {
+	/**
+	 * Post type name.
+	 *
+	 * @var string
+	 */
+	protected $object_type = 'category';
+	/**
+	 * Sitemap name
+	 * Used for building sitemap URLs.
+	 *
+	 * @var string
+	 */
+	public $name = 'categories';
+
+	/**
+	 * Sitemap route.
+	 *
+	 * Regex pattern used when building the route for a sitemap.
+	 *
+	 * @var string
+	 */
+	public $route = '^sitemap-categories-?([0-9]+)?\.xml$';
```

Comment:
Note: this does not yet match subtypes.

Reply:
Yeah, we'll need to adjust this for #22 and #49 but it's working for now and covers the acceptance criteria for this ticket.